public/data/nazmulhassan.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "MD NAZMUL HASSAN",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/nazmul19.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/nazmul19",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/nazmulhassan19",
+        "icon": "twitter"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Sharing and Learning",
+        "date": "December 2021",
+        "icon": "Globe",
+        "color": "grey",
+        "description": "Started Sharing and Learning"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

It can be done by gh-actions now 🤔